enhancements/single-node/single-node-openshift-day2-workers.md

```diff
@@ -0,0 +1,282 @@
+---
+title: single-node-openshift-day2-workers
+authors:
+  - "@omertuc"
+reviewers:
+  - "@romfreiman"
+  - "@eranco74"
+  - "@tsorya"
+  - TBD
+approvers:
+  - TBD
+api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers)
+  - TBD
+creation-date: 2021-01-05
+last-updated: 2021-01-05
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/MGMT-8414
+see-also:
+  - "https://github.com/openshift/enhancements/tree/master/enhancements/single-node"
+  - "https://github.com/openshift/enhancements/blob/master/enhancements/installer/single-node-installation-bootstrap-in-place.md"
+replaces:
+superseded-by:
+---
+
+# Single Node OpenShift Day-2 Workers
+
+## Summary
+
+This enhancemnet aims to enable adding workers to a single-node cluster by
+dealing with a "floating ingress" issue encountered in UPI-installed single
+control-plane node clusters which get worker nodes added to them. It does so by
+adjusting the installer to pin the default `IngressController` to the master
+pool when installing single-node bootstrap-in-place clusters.
```

Comment:
Why limit to bootstrap-in-place?

Reply:
It's mentioned a few times in the rest of the enhancement, but when installing IPI on e.g. AWS you don't suffer from this problem since you have a LoadBalancer type service that follows the node holding router-default automatically

Reply:
I see, so it's not about how you do the bootstrap it's about the platform.
Consider rephrasing `bootstrap-in-place` to None platform